javascript/javascript3/week2/Task3_Rewrite_Time/rewrite_time.js

```diff
@@ -0,0 +1,34 @@
+// TASK 3 Rewrite time
+
+/*Rewrite setTimeout and navigator.geolocation.getCurrentPosition to promises.
+ The getCurrentPosition function is probably going to throw an error, but that is fine.
+ As long as you can use it as a promise. */
+
+const setTimeoutPromise = (delay) => new Promise((resolve) => setTimeout(resolve, delay));
+
+setTimeoutPromise(3000).then(() => {
+  console.log("Called after 3 seconds");
+});
+
+const getCurrentLocation = () => {
+  return new Promise((resolve, reject) => {
```

Comment:
You can try adding code for the case when the request fails and therefore you can call `reject`

Reply:
Reject case has been added to the getCurrentLocation function.